### PR TITLE
update application.yml for namespace.

### PR DESCRIPTION
**Pull request by JaredTan95**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [x] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

**Comment by wu-sheng** [comment]
Does this work? Tested? If so, put it in assemble `application.yml`.

File: oap-server/server-starter/src/main/assembly/application.yml [hunk 1]
```diff
@@ -59,6 +60,7 @@ storage:
 #    bulkSize: ${SW_STORAGE_ES_BULK_SIZE:20} # flush the bulk every 20mb
 #    flushInterval: ${SW_STORAGE_ES_FLUSH_INTERVAL:10} # flush the bulk every 10 seconds whatever the number of requests
 #    concurrentRequests: ${SW_STORAGE_ES_CONCURRENT_REQUESTS:2} # the number of concurrent requests
+#  mysql:
```
Comment (peng-yongsheng) [inline oap-server/server-starter/src/main/assembly/application.yml]: Here is a mysql module config, but no parameter. Please confirm about it.

Comment (JaredTan95) [reply oap-server/server-starter/src/main/assembly/application.yml]: According to https://github.com/apache/incubator-skywalking/pull/1964 pr,mysql config would be placed in datasource-settings.properties file.Also,I'll make a fit for the file as soon as possible about the feature that override configuration through environment variables.

Comment (wu-sheng) [reply oap-server/server-starter/src/main/assembly/application.yml]: Yes. MySQL is based on the old setting file style, which is different. :) We could choose to move them into application.yml. 

Anyway, that is another feature to discuss.

Comment (JaredTan95) [reply oap-server/server-starter/src/main/assembly/application.yml]: yep,if move them into ` application.yml` , the setting file style will appear to be more uniform.:P

**Comment by wu-sheng** [comment]
And you need to put a document for backend namespace, which only in 5.x for now.

**Comment by wu-sheng** [comment]
I doubt the namespace works for cluster management module. Zookeeper cluster management has this setting?

**Comment by wu-sheng** [comment]
If only elastic search supports namespace, it only provide index name prefix. That is all.

**Comment by JaredTan95** [comment]
> I doubt the namespace works for cluster management module. Zookeeper cluster management has this setting?

As far as i kown, Zk has not this settings in skywalking collector.right?

**Comment by wu-sheng** [comment]
I mean, the document `/backend/namespace.md` is not so right. I understand you copy and change from v5.x, but it doesn't fit here. You could just say in `backend/backend-storage.md`

> When namespace is set, names of all indexes in ElasticSearch use the as prefix.

**Comment by JaredTan95** [comment]
aha, got it.

**Pull request merged**